README_libf3d.md

```diff
@@ -206,6 +208,31 @@ eng.getLoader().addFile("f3d/testing/data/dragon.vtu").loadFile()
 eng.getInteractor().start()
 ```
 
+## Java Bindings
+
+If the Java bindings are generated, the libf3d can be used directly from Java.
+You can import the `f3d.jar` package and use the provided Java classes directly.
+Make sure to set `java.library.path` to path where the JNI library is built.
+Here is an example showing how to use libf3d Java bindings:
```

Comment:
in the PR descr there is some info that could be added here

Reply:
The other comments in my PR are related to our CI infrastructure, I don't think it's relevant to add it to the documentation.

Reply:
ok